wordpress-mobile/WordPress-iOS
# Jetpack landing screen: extend background gradient to the whole view and add blur effect to the buttons

Fixes #19229 
Fixes #19228

This PR adds a blur effect to the prologue buttons, and extend the background gradient to the whole view

Related WordPressAuthenticator [PR](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/676)

https://user-images.githubusercontent.com/34376330/188240861-b7eddd87-fcc1-451b-9fc3-26a4ed8d46b2.mov


To test:

- checkout this branch and enable the `newLandingScreen` feature flag
- run `rake dependencies` to make sure the proper version of WordPressAuthenticator is pulled in
- Run the `Jetpack` target
- Log out if you're logged in
- Make sure the blur effect behind the buttons is visible. On iPad, it should extend along the whole width.

## Regression Notes
1. Potential unintended areas of impact
Prologue UI

2. What I did to test those areas of impact (or what existing automated tests I relied on)
test prologue with feature flag enabled/disabled on both targets (`Jetpack` and `WordPress`)

3. What automated tests I added (or what prevented me from doing so)
none
PR submission checklist:
